stores/postgres/ride_sql.go

```diff
@@ -27,7 +27,7 @@ CREATE TABLE IF NOT EXISTS rides (
 
 	rideGetAllSQL = "SELECT * FROM rides WHERE id > $1 LIMIT $2"
 
-	rideGetByIDSQL = "SELECT * FROM rides WHERE id=$1"
+	rideGetByIDSQL = "SELECT *, (SELECT COUNT(*) FROM passengers WHERE ride_id=$1 AND status='accepted') AS seats_taken FROM rides WHERE rides.id=$1;"
 
```

Comment:
where arey ou creating the passengers table?

Reply:
Its in the passenger_sql.go file